### DISCUSSION TITLE
No encuentro la pauta del informe

### DISCUSSION THREAD

**Discussion by tariztia** [IIC2133-PUC/2024-1#170] [Tarea 2] [answered]
Holaa, estuve buscando la pauta del informe pero no la encuentro. No está en la carpeta de pautas del repositorio. Donde podria encontrarla?
Graciass

**Comment by sebacarrascop** [comment, marked as answer]
hola, la pauta se entrega después, la idea es que el informe actúe como un documento de diseño que incluya los elementos solicitados en la sección de informe del enunciado de la tarea.

**Reply by tariztia** [reply]
Lo entiendo para la parte del desarrollo y la conclusión. Pero no se que debo poner en la parte de motivación, y me ha perjudicado en los otros informes. Si no hay pauta entonces me podrias explicar un poquito que debo poner por favor?
